### PR TITLE
 Clean up reference manual section on QEMU images

### PR DESCRIPTION
**Pull request by kprosise**
As part of regular documentation maintenance, tidied up the pages and
templates, changed admonitions where appropriate, applied semantic newlines,
removed words that could be removed. No major changes to content made.

For QA: ran linkcheck, built and viewed content for intended rendering.

It may be wise to validate steps for each QEMU machine as part of a
separate task.

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for de6fcca60bce71c6061940bd932103b77beed855 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1446/docs/artifacts/html/index.html

**Review by vanmaegima — APPROVED** [review]
Minor comments but LGTM

**Comment by doanac** [comment]
Docs for eb9cd6491f77cedcea9e124d52e5b11dd588d721 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1448/docs/artifacts/html/index.html

File: source/reference-manual/qemu/qemu-instructions.template [hunk 1]
```diff
@@ -9,31 +9,33 @@
...
-#. Make a directory to store the artifacts and ``cd`` into it.
+#. Make a directory the artifacts and ``cd`` into it:
 
    .. parsed-literal::
 
           mkdir -p lmp-qemu/|ARCH|
           cd lmp-qemu/|ARCH|
 
-#. Download the **necessary artifacts** for |ARCH|, replace ``<target_number>`` with the target you would like to boot
+#. Download the **necessary artifacts** for |ARCH|, replacing ``<target_number>`` with the target you would like to boot:
```
Comment (mwasilew) [inline source/reference-manual/qemu/qemu-instructions.template:32]: "necessary artifacts" were replaced with "artifacts" in line 1, but not here. Looks inconsistent to me.

File: source/reference-manual/qemu/qemu-instructions.template [hunk 2]
```diff
@@ -42,25 +44,22 @@ Booting in QEMU
...
-   ``lmp-qemu/``. You may want to save this as ``run.sh`` inside the
-   directory for convenience
+#. Run the QEMU script below against the artifacts inside of ``lmp-qemu/``.
+   You can save this as ``run.sh`` inside the directory for convenience.
 
 .. note::
-    The QEMU CLI passes the necessary flags and parameters to the
-    appropriate qemu-system command, such as image, CPU, network, and other device
-    information. For specifics, consult `QEMU's Documentation. <https://www.qemu.org/docs/master/index.html>`_
+    The QEMU CLI passes the necessary flags and parameters to the appropriate qemu-system command, such as image, CPU, network, and other device information.
```
Comment (mwasilew) [inline source/reference-manual/qemu/qemu-instructions.template:51]: This line looks too long. Shouldn't parts of the sentence be split into separate lines after comma and "and"?

Comment (kprosise) [reply source/reference-manual/qemu/qemu-instructions.template:51]: Yep, definitely too long.

File: source/reference-manual/qemu/qemu-instructions.template [hunk 3]
```diff
@@ -9,31 +9,33 @@
...
 
 .. note::
 	Make sure to set the FIOCTL_FACTORY environment variable:
 
-	export FIOCTL_FACTORY=<factory>
+    .. parsed-literal::
+
+        export FIOCTL_FACTORY=<factory>
+
+#. List available Targets and decide on which to boot::
```
[the remark this answers is not in the document]
Comment (kprosise) [reply source/reference-manual/qemu/qemu-instructions.template:21]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Yeah, it gets a bit confusing, I frequently have to check what is going on when it comes to blocks in rst/Sphinx.

**Comment by doanac** [comment]
Docs for 3916d2c2b6604a5a997afc5eeedf0a3dc4a9f847 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1452/docs/artifacts/html/index.html

**Pull request merged**